model/BUILD.bazel

```diff
@@ -149,7 +149,8 @@ proto_library(
 
 java_lite_proto_library(
     name = "voiceover_java_proto_lite",
-    deps = [":voiceover_proto"],
+    visibility = ["//visibility:public"],
```

Comment:
Try removing the visibility here and rebuild, since this proto is indirectly being exported by other protos. If it still builds, we don't need to expose the proto here as well.